lib/components/member_info_row.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:seeds/components/profile_avatar.dart';
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/utils/cap_utils.dart';
+
+class MemberInfoRow extends StatelessWidget {
+  final MemberModel member;
+
+  const MemberInfoRow({
+    Key? key,
+    required this.member,
+  }) : super(key: key);
```

Comment:
Extra comma, and no need to have a named parameter since it's just one.
` const MemberInfoRow( this.member, {Key? key}) : super(key: key);`